ambry-replication/src/test/java/com.github.ambry.replication/BlobIdTransformerTest.java

```diff
@@ -275,6 +320,10 @@ private void assertException(TransformationOutput transformationOutput, Class ex
         exceptionClass.isInstance(transformationOutput.getException()));
   }
 
+  public enum ChunkTypes {
```

Comment:
could you add documentation on what this class means and each of the enums?

Reply:
removed enum